package-lock.json

```diff
@@ -7217,14 +7216,24 @@
             "string-width": "^2.0.0",
             "which-module": "^2.0.0",
             "y18n": "^3.2.1 || ^4.0.0",
-            "yargs-parser": "^11.1.1"
+            "yargs-parser": "^13.1.2"
+          },
+          "dependencies": {
+            "yargs-parser": {
+              "version": "13.1.2",
```

Comment:
is this auto generated? as the version is different to the resolved url version number. Not sure what the resolved does but should they be the same?